pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -127,6 +127,11 @@ func (o *OAuthAPIServerOptions) NewOAuthAPIServerConfig() (*apiserver.Config, er
 	}
 
 	serverConfig := apiserver.NewConfig()
+
+	if err := o.GenericServerRunOptions.ApplyTo(&serverConfig.GenericConfig.Config); err != nil {
```

Comment:
would `github.com/openshift/library-go/pkg/config/configdefaults.SetRecommendedGenericAPIServerConfigDefaults` work?

Reply:
no, `GenericServerRunOptions.ApplyTo` is applying options - get from the command lines arguments to a configuration. We should respect command line options set by the operator or administrator.

Reply:
oh, that's a nasty bug we've got here then :eyes: